plugin/src/main/java/io/fabric8/maven/plugin/mojo/build/ResourceMojo.java

```diff
@@ -225,6 +225,9 @@
     @Parameter(property = "fabric8.namespace")
     private String namespace;
 
+    @Parameter(property = "fabric8.sidecar")
+    private String sidecar;
```

Comment:
It should be a boolean with default to false instead of String and parse it later.